make build failure

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://Rook-io.slack.com).
3. Did you already search the existing open issues for anything similar? -->

Is this a bug report or feature request?
<!-- Remove only one -->
* Bug Report


**Bug Report**

What happened:
"make build" failed on a t2.micro ubuntu

What you expected to happen:
everything built

How to reproduce it (minimal and precise):
<!-- Please let us know any circumstances for reproduction of your bug. -->
t2.micro, ubuntu, master branch from rook.

sha256:5c0f368fb38b97f9fb64b62a60c82a301c3c6df1b3ac787ac1bc77f8e26eafbf
=== caching image build-8a9e159f/base-amd64
=== caching image build-8a9e159f/rootfs-builder-amd64
=== caching image build-8a9e159f/rootfs-amd64-45d018cc0281
=== docker build build-8a9e159f/cross-gnu-amd64
sha256:62cabbf0d6f9c9f36e4b4049a30ac35728919f59694c8e18577bdded8d9efc70
=== caching image build-8a9e159f/cross-gnu-amd64
=== docker build build-8a9e159f/ceph-builder-amd64-tcmalloc-release
sha256:5689149571537f1ed41a57ab9ef05ccaa562c9b67fde53f1489638e1a6a3eec1
=== building ceph from build-8a9e159f/ceph-builder-amd64-tcmalloc-release
make: the '-j' option requires a positive integer argument
Usage: make [options] [target] ...
Options:
  -b, -m                      Ignored for compatibility.
  -B, --always-make           Unconditionally make all targets.
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
  -d                          Print lots of debugging information.
  --debug[=FLAGS]             Print various types of debugging information.
  -e, --environment-overrides
                              Environment variables override makefiles.
  --eval=STRING               Evaluate STRING as a makefile statement.
  -f FILE, --file=FILE, --makefile=FILE
...
...
...
  --warn-undefined-variables  Warn when an undefined variable is referenced.

This program built for x86_64-pc-linux-gnu
Report bugs to <bug-make@gnu.org>
Makefile:75: recipe for target 'ceph' failed
make[4]: *** [ceph] Error 2
../image.mk:134: recipe for target 'cache.lookup' failed
make[3]: *** [cache.lookup] Error 2
Makefile:59: recipe for target 'do.build' failed
make[2]: *** [do.build] Error 2
Makefile:30: recipe for target 'ceph.linux_amd64' failed
make[1]: *** [ceph.linux_amd64] Error 2
Makefile:113: recipe for target 'build' failed
make: *** [build] Error 2

**Environment**:
* OS (e.g. from /etc/os-release): Ubuntu 16.04.3 LTS (Xenial Xerus)
* Kernel (e.g. `uname -a`): Linux ip-172-31-30-89 4.4.0-1044-aws #53-Ubuntu SMP Mon Dec 11 13:49:57 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
* Cloud provider or hardware configuration: AWS
* Rook version (use `rook version` inside of a Rook Pod): rook: v0.6.0-102.g9678d80
* Kubernetes version (use `kubectl version`): N/A
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift): N/A
* Ceph status (use `ceph health` in the [Rook toolbox] (https://Rook.io/docs/Rook/master/toolbox.html)): N/A

--- a/images/ceph/build-ceph.sh
+++ b/images/ceph/build-ceph.sh
@@ -17,7 +17,12 @@
 CEPH_INSTALL_DIR=${CEPH_INSTALL_DIR:-/build/ceph-install}
 CEPH_TARBALL=${CEPH_INSTALL_DIR}.tar
 
-makeargs=-j$(( $(nproc) - 1 ))
+NPROC=$(( $(nproc) - 1 ))
+if [ "$NPROC" -eq 0 ]; then
+    NPROC=1
+fi
+
+makeargs=-j$NPROC
 
 # build and install
 make ${makeargs} rook

